ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```diff
@@ -302,17 +302,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb
-- as: aws-ipi-ingress-lbtype-nlb-arm-f2
+- as: aws-ipi-ingress-lbtype-nlb-mixd0-f2
```

Comment:
Compared with the name convention for our existing heterogeneous cluster, e.g: https://github.com/openshift/release/blob/0bb7977f410367a7865e7b96ff4ef159019a7e03/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml#L186, `amd-mixarch`, the `amd` stands for master arch, `mixarch` for compute arch, so can we keep the same convention, if that, this job name may like `aws-ipi-ingress-lbtype-nlb-arm-amd-day0-f2`, and update the `amd-mixarch` to `amd-mixarch-day2`, wdyt?

Reply:
done